### PR TITLE
docs(tutorial): Fix imports 

### PR DESCRIPTION
**Pull request by jessmartin**
### Summary
📦🛠️🧹

Updated tutorial code to use modular DXOS React client packages. This makes the code more consistent with the latest library version and easier to maintain.

> _We're sailing on the DXOS React client_
> _We've got to update our imports right_
> _So heave away, me hearties, on the count of three_
> _And switch to `halo` and `echo` for quality_

### Walkthrough
*  Update imports of `useIdentity` hook to use `@dxos/react-client/halo` package instead of deprecated `@dxos/react-client` package ([link](https://github.com/dxos/dxos/pull/3848/files?diff=unified&w=0#diff-866c56a2aceb95b0d17b1047327727487a7448580cf1f37666d683cba381f889L104-R110), [link](https://github.com/dxos/dxos/pull/3848/files?diff=unified&w=0#diff-866c56a2aceb95b0d17b1047327727487a7448580cf1f37666d683cba381f889L151-R153), [link](https://github.com/dxos/dxos/pull/3848/files?diff=unified&w=0#diff-866c56a2aceb95b0d17b1047327727487a7448580cf1f37666d683cba381f889L179-R182))
*  Update imports of `useSpaces`, `useQuery`, and `Expando` to use `@dxos/react-client/echo` package instead of deprecated `@dxos/react-client` package ([link](https://github.com/dxos/dxos/pull/3848/files?diff=unified&w=0#diff-866c56a2aceb95b0d17b1047327727487a7448580cf1f37666d683cba381f889L104-R110), [link](https://github.com/dxos/dxos/pull/3848/files?diff=unified&w=0#diff-866c56a2aceb95b0d17b1047327727487a7448580cf1f37666d683cba381f889L125-R126), [link](https://github.com/dxos/dxos/pull/3848/files?diff=unified&w=0#diff-866c56a2aceb95b0d17b1047327727487a7448580cf1f37666d683cba381f889L151-R153), [link](https://github.com/dxos/dxos/pull/3848/files?diff=unified&w=0#diff-866c56a2aceb95b0d17b1047327727487a7448580cf1f37666d683cba381f889L179-R182))

### CODE REVIEW

**Comment by jessmartin** [comment]
I noticed that the imports were already corrected in the `/snippets` directory. Is there a way to avoid duplicating them in the markdown file so that they are in sync? cc: @zhenyasav 

I guess I could go check the vuepress docs.

File: docs/docs/api/@dxos/config/interfaces/Kube.md [hunk 1]
```diff
@@ -5,14 +5,18 @@ Defined in:
...
-### [appTunneling]()
-Type: <code>[AppTunneling](/api/@dxos/config/interfaces/AppTunneling)</code>
+### [publicUrl]()
+Type: <code>string</code>
 
-### [autoupdate]()
-Type: <code>[Autoupdate](/api/@dxos/config/interfaces/Autoupdate)</code>
+Options:
+  - proto3_optional = true
+rfaces/Autoupdate)</code>
```
Comment (jessmartin) [inline docs/docs/api/@dxos/config/interfaces/Kube.md:19]: these changes resulted from running `nx apidoc`

File: docs/docs/guide/getting-started.md [hunk 2]
```diff
@@ -123,15 +121,15 @@ This will start the development server and print a URL to the console. Opening t
...
+*   Remote peers exchange data directly, **peer-to-peer** over secure [WebRTC](https://webrtc.org/) connections.
+*   User identity (public/private keys) are established securely and maintained by [HALO](./platform/halo) for the whole device (browser profile), without a password.
+*   Everything works offline.
+*   Real-time collaboration is possible when online.
+*   There are **no servers** that store any data.
+*   There is no need for [ORMs](https://en.wikipedia.org/wiki/Object%E2%80%93relational_mapping). ECHO objects are "plain javascript" objects that can be manipulated directly.
+*   **There is no need for an API tier.** The app has everything it needs on the client.
```
Comment (jessmartin) [inline docs/docs/guide/getting-started.md:132]: apologies for these formatting changes! working fast here.

File: docs/docs/guide/getting-started.md [hunk 3]
```diff
@@ -30,7 +30,7 @@ npm install --save @dxos/client
 
 Create and initialize a [`Client`](/api/@dxos/client/classes/Client):
 
-```ts file=./snippets/create-client.ts#L5-
+```ts file=./typescript/snippets/create-client.ts#L5-
```
Comment (jessmartin) [inline docs/docs/guide/getting-started.md:33]: this path was wrong, so `apidoc` was failing on this file

File: docs/docs/guide/getting-started.md [hunk 4]
```diff
@@ -60,15 +61,12 @@ npm install --save @dxos/react-client
 
 Create a `ClientProvider` to wrap your application. This allows nested components to use the hooks.
 
-```tsx file=./snippets/create-client-react.tsx#L5-
+```tsx file=./react/snippets/create-client-react.tsx#L5-
```
Comment (jessmartin) [inline docs/docs/guide/getting-started.md:64]: this path was wrong, so `apidoc` was failing on this file

**Review by wittjosiah — APPROVED** [review]

**Pull request merged**